tsconfig.json

```diff
@@ -42,7 +42,7 @@
     /* Strict Type-Checking Options */
     "strict": true /* Enable all strict type-checking options. */,
     /* Raise error on expressions and declarations with an implied 'any' type. */
-    // "noImplicitAny": true,
+    "noImplicitAny": false,
```

Comment:
Was this only used for debugging, or do we want to disable it entirely?

Reply:
I was running into an error that I can't reproduce 🤔   if it surfaces again, I'll make a more informative issue. Will revert this change.

Reply:
Got it. In your revert this line got commented out, but with the new value. To save another round-trip of reviewing & fixing I did a follow up which reverts the change proper: https://github.com/radicle-dev/radicle-upstream/pull/951/commits/5942685d054c07a5bccd10f190338938f24738bf.